src/pynxtools/dataconverter/convert.py

```diff
@@ -284,28 +284,32 @@ def main_cli():
     default=[],
     multiple=True,
     help=(
-        "Deprecated: Please use the positional file arguments instead. "
-        "The path to the input data file to read. (Repeat for more than one file.)"
+        "Deprecated: Please use the positional file arguments instead. The path to the "
+        "input data file to read. Repeat for more than one file. default=[] This option "
+        "is required if no '--params-file' is supplied."
     ),
 )
 @click.option(
     "--reader",
-    default="json_map",
+    default=None,
```

Comment:
I removed it because no reader should be preferred here

Reply:
It's not really preferred. That reader is added as the default behavior for using mapping. We added this to the docs before as well. In the future this can be replaced with the multiformat reader if it fits better. 

If we remove this we cannot directly use the mapping flag. These were quality of life improvements we did before.

```suggestion
    default="json_map",
```

Reply:
Agreed that it should be removed. The multiformat reader would not fit better because it does not actually implement the json mapping. I guess `json_map` could inherit from the multiformat reader eventually.

Reply:
Reverted